packages/hurl/src/cli/options/commands.rs

```diff
@@ -284,6 +284,16 @@ pub fn max_time() -> clap::Arg {
         .num_args(1)
 }
 
+pub fn max_filesize() -> clap::Arg {
```

Comment:
This file is generated (you should look at the begin of the file). To add a new option, you will have to look at [/docs/spec/options/README.md](https://github.com/Orange-OpenSource/hurl/blob/master/docs/spec/options/README.md)